server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -174,6 +173,11 @@ def _validate_and_initialize(self):
 
     @requires_data
     def _validate_data_types(self):
+        if sparse.isspmatrix(self.data.X) and not sparse.isspmatrix_csc(self.data.X):
+            warnings.warn(
+                f"Scanpy data matrix is sparse, but not a CSC (columnar) matrix.  "
+                f"Performance may be improved by using CSC."
```

Comment:
Can we point them to a reference for how to convert between the two? I suspect some biologists (like me, up until recently :) may not be super familiar with the nuances.

Reply:
that would be something for docs/FAQ, right?   What would you propose beyond the current advice in our FAQ, which points people at using `--sparse`?